### PR TITLE
Site updates for v3.2.1

### PR DESCRIPTION
**Pull request by parkr**
/cc @jekyll/core #5159

### CODE REVIEW

**Comment by parkr** [comment]
LGTM, I guess.

File: Gemfile [hunk 1]
```diff
@@ -63,11 +63,11 @@ group :jekyll_optional_dependencies do
...
-  gem "jekyll-gist", "~> 1.0"
-  gem "jekyll-feed", "~> 0.1.3"
-  gem "jekyll-coffeescript", "~> 1.0"
-  gem "jekyll-redirect-from", "~> 0.9.1"
-  gem "jekyll-paginate", "~> 1.0"
+  gem "jekyll-gist"
+  gem "jekyll-feed"
+  gem "jekyll-coffeescript"
+  gem "jekyll-redirect-from"
+  gem "jekyll-paginate"
```
Comment (mattr-) [inline Gemfile]: Is there a reason we don't want to pin these anymore?

**Comment by mattr-** [comment]
:-1: until the builds pass.

**Comment by parkr** [comment]
> 👎 until the builds pass.

@mattr- Would you take a look at #5188 for me? It worked when I applied that patch to this branch locally. 😄

File: Gemfile (hunk 1 above)
Comment (parkr) [inline Gemfile]: :wave: @mattr-!!

A couple of them are used in our tests but are not runtime dependencies. By pinning them, we open ourselves up to issues where we release a new version of one of these, say jekyll-paginate, and we don't catch that it is broken in these tests as we upgrade this code (e.g. we could break an API)

**Comment by parkr** [comment]
LGTM. Travis is passing.

File: Gemfile (hunk 1 above)
Comment (mattr-) [inline Gemfile]: :+1:

**Pull request merged**